charmcraft.yaml

```diff
@@ -13,5 +13,4 @@ parts:
   charm:
     prime:
       - files/*yaml
```

Comment:
Should we also remove this? `files` does not exist anymore and `templates` gets pushed automatically.

Reply:
This sounds like a candidate for a separate PR, I can make the change, but I wouldn't want this to get buried in the squash of this commit and gets lost if someone wants to check when this change was made.